lib/susedistribution.pm

```diff
@@ -96,14 +96,17 @@ sub x11_start_program($$$) {
         send_key('alt-t');
         sleep 3;
     }
-    send_key('ret', 1);
+    send_key "ret";
+    wait_still_screen;
     # make sure desktop runner executed and closed when have had valid value
```

Comment:
I have 2 notes here:
1. no test use $options->{valid} = 0 (keep the option?)
2. desktop-runner-border needles exist only for kde

Why do we use this if block? I do not see point of closing desktop runner multiple times without bug report.